tests/artifacts_tests/test_gcs.py

```diff
@@ -31,7 +31,6 @@ def blob(self, blob_name: str) -> "MockBlob":
         return MockBlob(blob_name)
 
     def delete_blob(self, blob_name: str) -> None:
-        global _MOCK_BUCKET_CONTENT
```

Comment:
Although the dict itself is not a global variable, the contents are globally editable.

```python
D = {}

def add():
    D["a"] = 1

def delete():
    del D["a"]
```